Marlin/Marlin_main.cpp

```diff
@@ -596,6 +596,43 @@ void setup_powerhold() {
   #endif
 }
 
+/*
+  Enable 24V to fans, E-reg, Cartridge Holder, Cartridges
+  if no short circuit is detected on line
+  ~200us, blocking time
+  Requires serial to be enabled to generate error condition
+*/
+void enable_power_supply() {
+  // Turn on FAN first so that 24V does not have a path to 24V_SW through the Rambo. TODO: Disconnect these two circuits.
+  pinMode(FAN_CHASSIS_TOP_PIN, OUTPUT);
+  digitalWrite(FAN_CHASSIS_TOP_PIN, HIGH);
+  delay(6);  // Delay for line to settle. 
+  
+  analogRead(PS_MONITOR_PIN);  // Throw out first reading
+  int V_Monitor_Result = analogRead(PS_MONITOR_PIN);
```

Comment:
Specify uint16_t or whatever it should be (I'm assuming 16 bit to account for the 10-bit ADC).